tree_structure.py

```diff
@@ -1,56 +1,57 @@
 # importing all the libraries and packages
 from anytree import Node, RenderTree, AsciiStyle, PreOrderIter, LevelOrderIter
+from pandas import pandas as pd
 
-#DEclare the HB as root resource
-high_bond_root = Node("high_bond",url="high_bond")
+#Declare the HB as root resource
+high_bond_root = Node("high_bond",url="high_bond",api_response=pd.DataFrame)
```

Comment:
I don't quite understand why you send a function here to `api_response` and later on, you assign it to a dataframe instance. Does a string here work?

Reply:
I am initializing node as data frame.